conan/tools/system/pip_manager.py

```diff
@@ -0,0 +1,64 @@
+import venv
+import platform
+import os
+
+from conan.errors import ConanException
+from conan.tools.env.environment import Environment
+
+
+class PipEnv:
+    accepted_install_codes = [0]
+    accepted_check_codes = [0, 1]
+
+    def __init__(self, conanfile, folder=None):
+        self._conanfile = conanfile
+        self.env_name = f"pip_venv_{self._conanfile.name}"
+        self._env_dir = os.path.abspath(os.path.join(folder or self._conanfile.recipe_folder, self.env_name))
```

Comment:
Why are we defaulting this to `self._conanfile.recipe_folder`? shouldn't this go to the `build_folder`?